app/components/provider_interface/references_summary_message.rb

```diff
@@ -0,0 +1,27 @@
+module ProviderInterface
+  class ReferencesSummaryMessage < ViewComponent::Base
+    attr_reader :references
+
+    delegate :feedback_provided, :feedback_requested, to: :references
+
+    def initialize(references)
+      @references = references
+    end
+
+    def message
+      if feedback_provided.blank?
+        "The candidate #{I18n.t('provider_interface.references.requested_message', count: feedback_requested.count)}."
+      else
+        "The candidate #{I18n.t('provider_interface.references.received_message', count: feedback_provided.count)}#{other_requested_reference}."
+      end
```

Comment:
given `other_requested_reference` already tests for `feedback_requested.blank?`, i think this can be simplified to
```
"The candidate #{I18n.t('provider_interface.references.received_message', count: feedback_provided.count)}#{other_requested_reference}."
```

Reply:
I initially thought that, but the first message is different from the second one.

Reply:
so it is!